k8s/airflow-operator/manifest/application.yaml.template

```diff
@@ -0,0 +1,40 @@
+    marketplace.cloud.google.com/deploy-info: '{partner_id: "click-to-deploy-containers", product_id: "airflow-operator", parnter_name: "Google Click to Deploy"}'
+  labels:
+    app.kubernetes.io/name: "$name"
+spec:
+  type: Airflow Operator
+  componentKinds:
+  - group: apps/v1beta2
+    kind: Deployment
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: "$name"
+  version: latest
+  description: |-
```

Comment:
in beta version, many fields here have been moved under `descriptor`. Please update them

Reply:
done